Dockerfile

```diff
@@ -194,6 +194,14 @@ RUN echo "emulator" && \
     . /etc/jdk.env && \
     yes | $ANDROID_SDK_MANAGER "emulator" > /dev/null
 
+RUN echo "build cmake" && \
+    . /etc/jdk.env && \
+    yes | $ANDROID_SDK_MANAGER \
+        "cmake;3.22.1" \
+        "cmake;3.18.1" \
+        "cmake;3.10.2.4988404" \
+        "cmake;3.6.4111459" > /dev/null
```

Comment:
Do we really need multiple version of `cmake`? Maybe just install the latest is enough.

Reply:
I guess it depends on how gradle handles conflicts with the installed cmake and a specified version in the `build.gradle` file.